docs/whatsnew/7.2.rst

```diff
@@ -24,6 +24,15 @@ By the numbers:
 * X pull requests have been merged since v7.1
 * X distinct people have contributed code
 
+
+.. _whatsnew-7.2-manylinux-upgrade:
+
+``manylinux`` upgrade
```

Comment:
We usually don't put such things in What's New but I will defer to @astropy/astropy-project-release-team